src/components/Course.jsx

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import React from 'react';
+import { observer } from 'mobx-react';
+import { Draggable } from 'react-beautiful-dnd';
+import '../styles/objects.Course.scss';
+import '../styles/utilities.shadow.scss';
+import '../styles/utilities.center.scss';
+
+export const Course = observer( props => (
+  <Draggable
+    draggableId={props.course.id}
+    type="TERM"
+  >
+    {(provided, snapshot) => (
+      <div>
+        <li
+          {...provided.dragHandleProps}
+          ref={provided.innerRef}
+          className="course shadow center"
+          style={{
+            backgroundColor: props.colorScheme.find(
```

Comment:
How frequently does this `find` function get called? Does it only get called if the information for the specific course is altered?